vtexdocs/RapiDoc
# Feature/Breadcrumbs

#### What is the purpose of this pull request?

To add the breadcrumbs componente to RapiDoc, so that we can use it in the authentication section.

Documented [here](https://www.notion.so/vtexhandbook/Adicionar-breadcrumbs-nas-p-ginas-de-API-Reference-ac7dc916102244fb8a30607e1c5af5c1)

#### What problem is this solving?

The authentication section of the API Reference pages in the Developer's Portal should have breadcrumbs.

#### How should this be manually tested?

Go to [this page](https://deploy-preview-57--elated-hoover-5c29bf.netlify.app/docs/api-reference/catalog#overview) and try clicking different links in the sidebar, so that you can see different documentations. In each of them, go to the authentication section (currently "Request") and compare the breadcrumbs against Figma. The headers should be clickable, but they all take you to the same page for now. Try accessing documentations with long names, such as [this one](https://deploy-preview-57--elated-hoover-5c29bf.netlify.app/docs/api-reference/catalog#delete-/api/catalog/pvt/skuservicetypeattachment/-skuServiceTypeAttachmentId-), and check that the name is cropped in the breadcrumbs.

#### Screenshots or example usage

#### Types of changes
